scopus-import/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -1754,19 +1754,9 @@ private Contributor findContributorByName(String givenName, List<Contributor> co
     }
 
     private void checkCollaborationName(CollaborationTp collaboration, List<Contributor> contributors) {
-        var optionalContributor = findContributorBySequence(collaboration.getSeq(), contributors);
-        if (Integer.parseInt(collaboration.getSeq()) < contributors.size()) {
-            var contributor = optionalContributor.getFirst();
+        var contributor = findContributorByName(collaboration.getIndexedName(), contributors);
```

Comment:
rename checkCollaborationName to checkThatCollaboration has been mapped.

Reply:
or maybe just checkCollaboration